src/tabs/GetStarted.tab

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<CustomTab xmlns="http://soap.sforce.com/2006/04/metadata">
+    <description>Created by Lightning App Builder</description>
```

Comment:
I think we can update the description here to be more specific, such as "NPSP Get Started Page". Can you give that a try to see if it's updateable?